src/components/Navbar/Navbar.jsx

```diff
@@ -7,8 +7,11 @@ import ProfileDropdown from './ProfileDropdown';
 import logo from '../../assets/OCH_Logo_SVG.svg';
 import Toast from '../Toast';
 import NavbarMobile from './NavbarMobile';
+import { useUserContext } from '../../common/UserContext/UserContext';
 
-const Navbar = ({ isAdmin, onAdminPortal, setOnAdminPortal, changesMade }) => {
+const Navbar = ({ onAdminPortal, setOnAdminPortal, changesMade }) => {
+  const userData = useUserContext();
+  const isAdmin = userData.userData.role === 'admin';
```

Comment:
This has already been worked on so and merged.

You can either revert these changes or when you merge dev into your branch to resolve conflicts, you can accept the incoming change.